PubSubSpec.md

```diff
@@ -92,6 +92,17 @@ The following well-known additional blocks are RECOMMENDED.
 Timestamp = TimestampNameComponent
 ```
 
+### Name Mapping Delivery Optimization
+
+To optimize the delivery of mapping data, SVS-PS implementations MAY piggyback mapping data in the Sync Interest's Application Parameters. If present, the mapping data SHOULD be appended as a single block after the StateVector in the Application Parameters. The mapping data MUST be encoded as a TLV block of type `MappingData` as defined above.
+
+```
+Sync Interest
+Application Parameters: StateVector MappingData
+```
```

Comment:
I'm not sure how to read this... what is it supposed to represent? if it's the format, why not use ABNF syntax?

Although, if it's the format, then a question of ordering between the two TLVs arises: this seems to mandate a specific order, but the preceding text says "SHOULD".